### DISCUSSION TITLE
Schedule free with exponentially decaying lr

### DISCUSSION THREAD

**Discussion by jwson302** [PredictiveIntelligenceLab/jaxpi#35] [General]
I am not sure if here is the right place to ask this, but in the paper

"Simulating Three-dimensional Turbulence with Physics-informed Neural Networks"

exponential decay of learning rate is combined used with schedule free method.

However, to my knowledge, the intent of the schedule free method is to replace handcrafted scheduling, and documentation on optax.contrib.schedule_free also states that it is important to turn off momentum of the base optimizer. Looking at the JAXPI repository, both exponential decay and momentum is used alongside schedule free method.

Should there be a reason why schedule free method performs better with momentum and decaying lr schedule in the context of PINNs?
